source/docs/publish/integration.md

```
@@ -17,28 +17,31 @@ For an integration repository to be valid, it must meet the requirements below.
 ```
-custom_components/awesome/__init_.py
+custom_components/awesome/__init__.py
 custom_components/awesome/sensor.py
 custom_components/awesome/manifest.json
 README.md
+hacs.json
 ```
 
 #### Not OK example (1):
 
 ```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)